.gitignore

```diff
@@ -0,0 +1,261 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Please update this file to the latest version on [VisualStudio.gitignore](https://github.com/github/gitignore/blob/master/VisualStudio.gitignore)